_posts/en/newsletters/2019-11-27-newsletter.md

```diff
@@ -0,0 +1,280 @@
+
+    - *Deprecated or removed features:* support for the [BIP70][]
+      payment protocol, [BIP37][] P2P protocol bloom filters, and
+      [BIP61][] P2P protocol reject messages has been disabled by
+      default, eliminating the source of various problems (respectively,
+      see Newsletters [#19][news19 bip70], [#57][news57 bip37], and
+      [#37][news37 bip61]).  The payment protocol and reject messages
+      are scheduled to be fully removed in the next major Bitcoin Core
+      version about six months from now.
+
+    - *Customizable permissions for whitelisted peers:* when
+      specifying which peers or interfaces should be whitelisted, users
+      can now specify which special features the whitelisted peers
+      can access.  Previously, whitelisted peers wouldn't be banned and
+      received relayed transactions faster.  These defaults haven't
+      changed, but now it's possible to toggle those settings on a
+      per-peer basis or to allow specified whitelisted peers to request
+      BIP37 bloom filters even though they're disabled for
+      non-whitelisted peers by default.  For details, see [Newsletter
+      #60][news60 16248].
+
+    - *GUI improvements:* graphical users can now create new wallets
+      for use with multiwallet mode from the GUI's *file* menu (see
+      [Newsletter #63][news63 new wallet]).  The GUI also now provides
+      users with [bech32][topic bech32] Bitcoin addresses by default,
+      but users can easily request a backwards-compatible P2SH-P2WPKH
+      address by toggling a checkbox next to the button to generate an
+      address (see [Newsletter #42][news42 core gui bech32]).
+
+    - *Optional privacy-preserving address management:* a new
+      `avoid_reuse` wallet flag, which can be toggled using a new
+      `setwalletflag` RPC, can be enabled to prevent the wallet from
+      spending bitcoins received to an address that was previously
+      used (see [Newsletter #52][news52 avoid_reuse]).  This prevents
+      certain privacy leaks based on block chain analysis such as [dust
+      flooding][].
+
+    For a full list of notable changes, links to the PRs where those
+    changes were made, and additional information useful to node
+    operators, please see the Bitcoin Core project's [release notes][notes
+    0.19.0].
+
+- **New LND mailing list and new host of existing mailing lists:** a [new
+  mailing list][lnd engineering] hosted by Google Groups was announced
+  for LND application developers, with an [initial post][osuntokun lnd
+  plans] by Olaoluwa
+  Osuntokun describing short-term goals for LND's next release.
+  Separately, the existing mailing lists for [Bitcoin-Dev][] and
+  [Lightning-Dev][] have recently had their hosting [transferred][togami
+  ml update] to Oregon State University's [Open Source Lab][osl] (OSL), a
+  well-respected organization that provides hosting to a variety of open
+  source projects.  Optech extends our thanks to everyone involved in
```

Comment:
It might be a good opportunity to explicitly call out Bryan and Warren who put in a lot of work on the mailing list.